scripts/generate-schema-types.js

```diff
@@ -4,16 +4,24 @@ const { compileFromFile } = require('json-schema-to-typescript')
 const prettier = require('prettier')
 const { printLog, logAndExit } = require('./utils')
 
-const workingDirectory = path.join(__dirname, '../rum-events-format')
-const schemaPath = path.join(workingDirectory, 'rum-events-format.json')
-const compiledTypesPath = path.join(__dirname, '../packages/rum-core/src/rumEvent.types.ts')
+const schemasDirectoryPath = path.join(__dirname, '../rum-events-format/schemas')
 const prettierConfigPath = path.join(__dirname, '../.prettierrc.yml')
 
 async function main() {
+  await generateTypesFromSchema('../packages/rum-core/src/rumEvent.types.ts', 'rum-events-schema.json')
+  await generateTypesFromSchema(
+    '../packages/core/src/domain/internalMonitoring/telemetryEvent.types.ts',
+    'telemetry-events-schema.json'
+  )
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏we need to look at `generateTypesFromSchema` to understand that the types path are relative to `__dirname` (and not the CWD). IMO it would be easier to understand by building the path there, like:

```suggestion
  await generateTypesFromSchema(path.join(__dirname, '../packages/rum-core/src/rumEvent.types.ts'), 'rum-events-schema.json')
  await generateTypesFromSchema(
    path.join(__dirname, '../packages/core/src/domain/internalMonitoring/telemetryEvent.types.ts'),
    'telemetry-events-schema.json'
  )
```